pkg/adapter/network.go

```diff
@@ -106,42 +98,108 @@ func (r *LocalRuntime) NetworkRemove(cli *cliconfig.NetworkRmValues) error {
 	return nil
 }
 
-// getCNIConfigPathByName finds a CNI network by name and
-// returns its configuration file path
-func getCNIConfigPathByName(name, cniConfigPath string) (string, error) {
-	files, err := libcni.ConfFiles(cniConfigPath, []string{".conflist"})
+// NetworkCreate creates a CNI network
+func (r *LocalRuntime) NetworkCreate(cli *cliconfig.NetworkCreateValues) (string, error) {
+	var (
+		err error
+	)
+
+	isGateway := true
+	ipMasq := true
+	subnet := &cli.Network
+	gateway := cli.Gateway
```

Comment:
Don't you need to make sure that subnet is also set if this is set?

Reply:
it is checked in adapter.